src/preferInlineDecoratorRule.ts

```diff
@@ -102,3 +103,7 @@ export class PreferInlineDecoratorWalker extends NgWalker {
     this.addFailureAt(decoratorStartPos, property.getWidth(), getFailureMessage(), fix);
   }
 }
+
+function isSameLine(sourceFile: ts.SourceFile, pos1: number, pos2: number) {
```

Comment:
Any reason to not use `tsutils`?

Reply:
`tsutils` isn't a dependencies/peerDependencies of Codelyzer

Reply:
You're right, however it's a `tslint` dependency. Anyway, can't we add it to codelyzer dependencies?

Reply:
IMO, sometimes, it's safer to duplicate some code lines than to be linked to a third-party  library.

Reply:
In this case, it would be better to move this function to `util`.